live-editing/configs/GridConfigGenerator.ts

```diff
@@ -1224,6 +1224,16 @@ export class GridConfigGenerator implements IConfigGenerator {
             })
         }));
 
+        configs.push(new Config({
+            component: 'GridSelectComponent',
+            additionalFiles: ["/src/app/directives/prevent-scroll.directive.ts", "/src/app/grid/grid-select/data.ts", "/src/app/grid/grid-select/characters.ts"],
+            appModuleConfig: new AppModuleConfig({
+                imports: ['IgxPreventDocumentScrollModule', 'IgxGridModule', 'IgxSelectModule', 'IgxDialogModule', 'GridSelectComponent' ],
```

Comment:
As the igxDialog is not used in this sample, please remove the IgxDialogModule references as well as all dialog related stuff (incl. .ts & .scss files)

Reply:
Removed the reference igxDialog and its logic.